### PR TITLE
Make django-ace (toolbar) support dark mode better

### PR DESCRIPTION
**Pull request by Ninjaclasher**
Part of #2035.

Screenshots (dark):
image
image

Screenshots (light)
image
image

### CODE REVIEW

File: resources/ace-dmoj.scss [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    > div {
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+    }
+}
+
+.django-ace-widget.loading {
```
Comment (quantum5) [inline resources/ace-dmoj.scss:15]: Could turn this into proper Sass with `&.loading`.

File: resources/style.scss [hunk 2]
```diff
@@ -19,3 +19,4 @@
 @import "contest";
 @import "misc";
 @import "select2-dmoj";
+@use "ace-dmoj";
```
Comment (quantum5) [inline resources/style.scss:22]: Merge conflict.

File: make_style.sh [hunk 3]
```diff
@@ -20,7 +20,12 @@ build_style() {
   echo "Creating $1 style..."
   cp resources/vars-$1.scss resources/vars.scss
   sass resources:sass_processed
-  postcss sass_processed/style.css sass_processed/martor-description.css sass_processed/select2-dmoj.css --verbose --use autoprefixer -d $2
+  postcss \
+      sass_processed/style.css \
+      sass_processed/martor-description.css \
+      sass_processed/select2-dmoj.css \
+      sass_processed/ace-dmoj.css \
+      --verbose --use autoprefixer -d $2
```
Comment (quantum5) [inline make_style.sh:28]: Missing quotes around `$2`, can't unsee.

File: resources/ace-dmoj.scss [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+.django-ace-toolbar {
+    font-size: 12px;
+    text-align: left;
+    color: $color_primary66;
+    text-shadow: 0 1px 0 $color_primary0;
+    border-bottom: 1px solid $color_primary25;
+    background-image: -moz-linear-gradient(rgba(0,0,0,0), $color_primary10);
+    background-image: -webkit-linear-gradient(rgba(0,0,0,0), $color_primary10);
+    background-image: linear-gradient(rgba(0,0,0,0), $color_primary10);
```
[the remark this answers is not in the document]
Comment (Ninjaclasher) [reply resources/ace-dmoj.scss]: Looks like we can just rely on autoprefixer, removed.

File: resources/ace-dmoj.scss (hunk 1 above)
Comment (Ninjaclasher) [reply resources/ace-dmoj.scss]: Done

**Review by quantum5 — APPROVED** [review]

**Pull request merged**